pkg/oc/admin/policy/reconcile_clusterroles_test.go

```diff
@@ -4,20 +4,18 @@ import (
 	"testing"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/util/sets"
 	kapihelper "k8s.io/kubernetes/pkg/api/helper"
-
-	authorizationapi "github.com/openshift/origin/pkg/authorization/apis/authorization"
+	"k8s.io/kubernetes/pkg/apis/rbac"
 )
 
-func role(rules []authorizationapi.PolicyRule, labels map[string]string, annotations map[string]string) *authorizationapi.ClusterRole {
-	return &authorizationapi.ClusterRole{Rules: rules, ObjectMeta: metav1.ObjectMeta{Labels: labels, Annotations: annotations}}
+func role(rules []rbac.PolicyRule, labels map[string]string, annotations map[string]string) *rbac.ClusterRole {
+	return &rbac.ClusterRole{Rules: rules, ObjectMeta: metav1.ObjectMeta{Labels: labels, Annotations: annotations}}
 }
 
-func rules(resources ...string) []authorizationapi.PolicyRule {
-	r := []authorizationapi.PolicyRule{}
+func rules(resources ...string) []rbac.PolicyRule {
+	r := []rbac.PolicyRule{}
 	for _, resource := range resources {
-		r = append(r, authorizationapi.PolicyRule{Verbs: sets.NewString("get"), Resources: sets.NewString(resource)})
+		r = append(r, rbac.NewRule("get").Groups("testgroup").Resources(resource).RuleOrDie())
```

Comment:
Where did `testgroup` come from?

Reply:
``rbac.PolicyRule`` requires a group for resource policies. The test won't pass without a group. I came up with an arbitrary group name. Do we have preferred names for tests?

Reply:
Just build the `rbac.PolicyRule{}` directly instead of using `RuleOrDie`